modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -1665,7 +1714,10 @@ private CompletableFuture<TransactionResult> finishTransaction(
                                 txId,
                                 commit,
                                 commitTimestamp,
-                                catalogVersion
+                                catalogVersion,
+                                aggregatedGroupIds.stream()
+                                        .map(PartitionReplicaListener::tablePartitionId)
```

Comment:
Here and there: I'd rather skip using table in tablePartitionId because later it'll be zonePartitionId, so that, simple partitionId seems enough to me.